packages/tests/src.ts/test-providers.ts

```diff
@@ -1418,4 +1419,19 @@ describe("Test Hedera Provider", function () {
        // @ts-ignore
        assert.strictEqual(record.valid_duration_seconds, '120');
    });
+
+   it("Is able to get hedera provider as default", async() => {
```

Comment:
I think that we should add test for `ethers.providers.getDefaultProvider()` and verify that it connects to mainnet